src/styles/calendar.scss

```diff
@@ -0,0 +1,46 @@
+.calendar {
+  gap: $gap;
+  padding-top: 50px;
+  display: flex;
+  flex-wrap: wrap;
+  width: $calendar-width;
+
+  &__day {
+    @include square($day-size);
+    display: flex;
+    flex-wrap: wrap;
+    box-sizing: content-box;
+    background-color: #eee;
+    border: 1px black solid;
+    transition-duration: 0.5s;
+
+    &:hover {
+      transform: translateY(-20px);
+      cursor: pointer;
+      background-color: pink;
+    }
+  }
+
+  @for $date from 1 through 31 {
+    &__day:nth-child(#{$date})::before {
+      content: "#{$date}";
+      margin: auto;
+    }
+  }
+
+  @each $day, $number in $days {
+    &--start-day-#{$day} {
+      &__day:first-child {
+        margin-left: ($day-size * $number + $gap *$number + $padding);
+      }
```

Comment:
what is the purpose of padding here?
i think it could be just 
`($day-size + $gap) * $number`

Reply:
At first, I thought it ought to be the same code that you suggested, but it didn't work, so I started playing around with it until it finally got into place. But now, as you said, the loop doesn't seem to be having any effect on the days at all... Where did I go wrong? I'm so confused by it....